Documentation/JobManager/README.md

```diff
@@ -0,0 +1,45 @@
+# Job Manager
+
+Jobs by definition are long running operations and especially when using mobile devices, applications are paused, resumed and killed based on the rules they have in the platforms. Our jobs supports pause and resume workflows but we don't have easy way to implement persistence to the application lifecycles without quite a bit custom code.
+
+The Job Manager is a toolkit component that you can just plug into the application and then give it tasks you want to persist when the application is paused and an easy way to rehydrate when the application is resumed.
```

Comment:
I don't know if pause and resume are good ways of describing the application lifecycle. It's more of when an application is backgrounded and evicted/terminated, and re-launched.